### ISSUE TITLE
Proposal to introduce maximum contrast ratio

### ISSUE THREAD

**Issue by cawa-93** [w3c/wcag3#171]
> Success Criterion 1.4.6 Contrast (Enhanced)
> (Level AAA)
> 
> The visual presentation of text and images of text has a contrast ratio of at **least 7:1**

However, this also means that the contrast ratio can be much higher. **Here are common [example](https://codepen.io/cawa-93/pen/oNZyKaj)**: white text on black background.

![зображення](https://user-images.githubusercontent.com/1662812/121163232-23423080-c83e-11eb-99b4-abd9059de810.png)

This text fully meets all Contrast Criterion AAA level (Contrast Ratio for Me 21:1). However, it is absolutely not readable.

I propose to introduce the **maximum** along with the minimum contrast ratio.

**Comment by kepstin** [comment]
Since "black on white" specifically is very common, I've set my displays to an appropriate brightness level to provide a comfortable reading environment for that combination. My displays have white luminance calibrated to 150 cd/m², in room with typical illuminance around 100 lx (these are IPS LCD displays, white LED backlight, with a rated static contrast ratio of approx 1000:1).

Although I am comfortable with "black on white" at this level, "white on black" is still practically unreadable, in part because of halation due to astigmatism. I find Lc -90 to be uncomfortable - if I do have to read light text on a dark background, I'd prefer a maximum contrast of around … maybe Lc -50 or so?

In an environment like mine, having the author use lower contrast colours for dark text on light background can reduce readability even when the author's intent was to do the opposite. In the example image in https://github.com/w3c/wcag3/issues/171#issuecomment-2578412464 I actually find the second block of text, with blue text over a brown background, to be harder to read than the first block with black text over a white background (although the first block still has issues due to the thin font).

I agree that there are problems caused by people using screens at too-high brightness, both for people using dark text on light backgrounds and for people using light text on dark backgrounds. I'm just not sure what the best solution is when you want to improve readability for people using too-bright displays, while ensuring that you do not reduce contrast too much for people who have set their displays to brightness levels appropriate to their environment.
